website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -74,7 +74,7 @@ You may be able to manage the following organization permissions.
 
 #### General
 
-Review the organization name and contact email. Organization owners can also change the organization name and contact email or delete the organization.
+Review the organization name and contact email. Organization owners can choose to change the organization name, contact email, and the default execution mode, or delete the organization. The default execution mode will change the execution mode across all workspaces that are set to use default execution mode, it is initially set as "Remote".
```

Comment:
I think it's better to omit the extra details about default execution mode in this doc, and instead insert a link so they can go learn more:
```suggestion
Review the organization name and contact email. Organization owners can change the organization name, contact email, [default execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode), or delete the organization.
```

How does that feel to you?

Reply:
It feels valuable to me to put it here, because the concept of `default_exection_modes` exists at the organization level, and is slightly different than the regular execution mode setting. But I can change that last sentence to link back to workspaces settings page